test/integration/patients/patient/patient-flow.js

```diff
@@ -330,7 +330,7 @@ context('patient flow page', function() {
       .find('.comment__item')
       .last()
       .as('socketComment')
-      .should('contain', formatDate(testTs(), 'AT_TIME'))
+      .should('contain', formatDate(dayjs.utc().format(), 'AT_TIME'))
```

Comment:
Unrelated to this PR. I've noticed this has been causing flakiness for a little while now.

Reply:
yeah..  tests revolving things happening in real time and not on the api probably need to be like this.. _or_ we need `visitOnClock` where we're controlling what's happening, but yeah this matches that first line in `onPostNewComment`